src/main/java/com/scalar/db/benchmarks/tpcc/transaction/OrderStatusTransaction.java

```diff
@@ -60,60 +61,61 @@ public void generate() {
     }
   }
 
-  /**
-   * Executes the order-status transaction.
-   *
-   * @param manager a {@code DistributedTransactionManager} object
-   */
   @Override
-  public void execute(DistributedTransactionManager manager) throws TransactionException {
-    DistributedTransaction tx = manager.start();
-
-    try {
-      if (byLastName) {
-        if (config.useTableIndex()) {
-          customerId = TpccUtil.getCustomerIdByTableIndex(tx, warehouseId, districtId, lastName);
-        } else {
-          customerId =
-              TpccUtil.getCustomerIdBySecondaryIndex(tx, warehouseId, districtId, lastName);
-        }
-      }
-
-      // Get customer
-      Optional<Result> result = tx.get(Customer.createGet(warehouseId, districtId, customerId));
-      if (!result.isPresent()) {
-        throw new TransactionException("Customer not found");
-      }
+  public void execute() throws TransactionException {
+    transaction = manager.start();
 
-      // Find the last order of the customer
-      int orderId;
+    if (byLastName) {
       if (config.useTableIndex()) {
-        orderId = getOrderIdByTableIndex(tx);
+        customerId =
+            TpccUtil.getCustomerIdByTableIndex(transaction, warehouseId, districtId, lastName);
       } else {
-        orderId = getOrderIdBySecondaryIndex(tx);
+        customerId =
+            TpccUtil.getCustomerIdBySecondaryIndex(transaction, warehouseId, districtId, lastName);
       }
+    }
 
-      // Get order
-      result = tx.get(Order.createGet(warehouseId, districtId, orderId));
-      if (!result.isPresent()) {
-        throw new TransactionException("Order not found");
-      }
+    // Get customer
+    Optional<Result> result =
+        transaction.get(Customer.createGet(warehouseId, districtId, customerId));
+    if (!result.isPresent()) {
+      throw new TransactionException("Customer not found");
```

Comment:
Thank you for the comment. It's a good idea. However, can we add this in another PR because 1) this is not directly related to the Kepie and 2) we should not use `TransactionException` directly (cf. the interface changed a bit in ScalarDB 3.8), and it's good to update them in the same time?